defaults/main.yml

```diff
@@ -50,7 +50,7 @@ k8s_domain_from_to_www_redirect: false
 k8s_namespace: "echoserver"
 
 k8s_memcached_enabled: false
-k8s_memcached_version: "1.5.19"
```

Comment:
Looks reasonable to me 👍 

Are there any concerns when upgrading an existing deploy?

Reply:
They don't call out any significant breaking changes in the switch from 1.5 to 1.6, but they have this caveat at the end of the release notes.

https://github.com/memcached/memcached/wiki/ReleaseNotes160

> We believe this software to be of high quality. However this release is a bit larger than usual. If you have critical infrastructure on memcached, please roll this out slowly if possible

Reply:
Also, rebar-caktus trial was migrated from 1.5.19 to the current and we didn't see much happen. Granted we had just deployed 1.15.19 so there wasn't a lot going on there in the first place.

Reply:
In conclusion this has been deployed on two different projects and seems to stable and running without  side effects.